docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -90,3 +90,33 @@ You can provide any custom component to this slot as long as it passes `children
 In the examples below we provide ways of implementing date range shortcuts using this slot.
 
 {{"demo": "PaperContentComponent.js", "defaultCodeOpen": false, "disableAd": true}}
+
+## Tabs
+
+The tabs are available on all date time picker components.
+By default, it is `hidden` on desktop, and `visible` on mobile.
+This behavior can be overridden by setting `hideTabs` prop.
```

Comment:
Maybe for v6: Should we drop the `hideTabs` prop in favor of a `hide` prop to the `Tabs` slot that makes the component return null ?

Reply:
That could be quite a logical change. 🤔 
Decoupling the props from higher level/order components and changing them directly on the components in question. 👍